### PR TITLE
Document endpoints for the PoS module - Closes 1723

### LINKED ISSUE #1723: Document the endpoints for the `pos` module.
### Description
Document the endpoints for the `pos` module.

#### Affected branch / URL
`development`

#### Page location
`lisk-docs/api/`

### Additional resources
https://github.com/LiskHQ/lisk-sdk/blob/development/framework/src/modules/pos/endpoint.ts

### PR DESCRIPTION
**Pull request by TalhaMaliktz**
Closes #1723

### CODE REVIEW

File: docs/modules/ROOT/pages/api/module-rpc-api/pos-endpoints.adoc [hunk 1]
```diff
@@ -0,0 +1,2846 @@
...
+{
+  "amount": "100000000000"
+}
+----
+====
+
+
+
+== `pos_getConstants`
+Returns the constants used by the PoS module.
```
Comment (Tschakki) [inline docs/modules/ROOT/pages/api/module-rpc-api/pos-endpoints.adoc]:
```suggestion
Returns the configuration of the PoS module.
```

The constants in the response are set in the node config

File: docs/modules/ROOT/pages/api/module-rpc-api/pos-endpoints.adoc [hunk 2]
```diff
@@ -0,0 +1,2846 @@
...
+=====
+
+.Response
+[%collapsible]
+====
+.Example output
+[source,json]
+----
+{
+  "pendingUnlocks": []
```
Comment (Tschakki) [inline docs/modules/ROOT/pages/api/module-rpc-api/pos-endpoints.adoc]: If possible, replace this with a non-empty response

File: docs/modules/ROOT/pages/api/module-rpc-api/pos-endpoints.adoc [hunk 3]
```diff
@@ -0,0 +1,2846 @@
...
+=====
+
+.Response
+[%collapsible]
+====
+.Example output
+[source,json]
+----
+{
+  "rewards": []
```
Comment (Tschakki) [inline docs/modules/ROOT/pages/api/module-rpc-api/pos-endpoints.adoc]: If possible, replace this with a non-empty response

**Review by Tschakki — CHANGES_REQUESTED** [review]
Looking good already 👍

**Review by Tschakki — APPROVED** [review]
Thank you!

**Review by mosmartin — APPROVED** [review]

**Pull request merged**